Re-skin and restructure the DEA usage guide

omad opened the pull request:
View a preview of the changes at https://damien-dea-docs.netlify.com/

- Re-skin with a DEA appropriate colour scheme
- Use a scaleable logo which looks much better on high res screens
- Make all the awesome notebook/user guides more discoverable

omad committed:
Documentation updates

omad committed:
Re-theme the documentation site

- Fix up the logo to be cleaner and sharper (use SVG)
- Use DEA colours instead of the default RTD theme

andrewdhicks commented on docs/index.rst:
```diff
@@ -39,20 +39,22 @@ Publicly available data access and web services are currently in development.
+   notebooks/Workflows/README.rst
+   notebooks/Scripts/README.rst
 
-   notebooks/index
```
I'm uncomfortable with including files so deep into a different repo, but I couldn't find a way to get it to work with toctree.

omad replied:
I was trying to do a:
```rst
.. include:: notebooks/index.rst
```
But it doesn't work nicely due to relative paths. We could change that file to specify the path correctly, and then be updated, but, that's a little yucky and wouldn't be viewable directly there.

I also tried doing
```rst
.. toctree::
   :glob:
   :caption: 

   notebooks/*/README.rst
```
But this will include the sections out of order. A possible option would be to rename the folders in the `dea-notebooks` directory.

Thoughts?

omad committed:
More docs cleaning

- Make the page background white, following the DEA style guide and logo bg
- Rely on the renumbered directories in the dea-notebooks repo
- Remove and reorder some of the internal docs

omad commented:
Okay, I've used the globbing feature of `toctree` and have renamed the directories in the `dea-notebooks` repo to keep the chapters in order. See https://github.com/GeoscienceAustralia/dea-notebooks/pull/183

andrewdhicks reviewed (approved):
Looks great!

omad committed:
Try to unbreak Travis YAML
